sdk/quantum/Azure.Quantum.Jobs/samples/Program.cs

```diff
@@ -26,7 +28,7 @@ public static void Main(string[] args)
             var location = "your_location";
             var storageContainerName = "your_container_name";
 
-            var credential = new DefaultAzureCredential(true);
+            TokenCredential credential = new DefaultAzureCredential(true);
```

Comment:
Why?

Reply:
I changed it when manually testing it, but will revert it to `var`.